src/NServiceBus.RabbitMQ/AssemblyInfo.cs

```diff
@@ -1,9 +1,3 @@
-﻿using System.Reflection;
-using System.Runtime.InteropServices;
+﻿using System.Runtime.InteropServices;
 
-[assembly: AssemblyTitle("NServiceBus.RabbitMQ")]
-[assembly: AssemblyCopyright("Copyright NServiceBus. All rights reserved")]
-[assembly: AssemblyProduct("NServiceBus.RabbitMQ")]
```

Comment:
Because the csproj does not have the equivalent elements for `AssemblyTitle` and `AssemblyProduct`, they are defaulting to:

```C#
// from ILSpy
[assembly: AssemblyProduct("NServiceBus.Transports.RabbitMQ")]
[assembly: AssemblyTitle("NServiceBus.Transports.RabbitMQ")]
```

So, a change from what they currently are. Does that matter?